Packages/com.unity.inputsystem/InputSystem/Plugins/InputForUI/InputSystemProvider.cs

```diff
@@ -565,39 +565,43 @@ void RegisterActions(Configuration cfg)
         {
             m_InputActionAsset = cfg.ActionAsset;
 
-            m_PointAction = InputActionReference.Create(m_InputActionAsset.FindAction(m_Cfg.PointAction));
-            m_MoveAction = InputActionReference.Create(m_InputActionAsset.FindAction(m_Cfg.MoveAction));
-            m_SubmitAction = InputActionReference.Create(m_InputActionAsset.FindAction(m_Cfg.SubmitAction));
-            m_CancelAction = InputActionReference.Create(m_InputActionAsset.FindAction(m_Cfg.CancelAction));
-            m_LeftClickAction = InputActionReference.Create(m_InputActionAsset.FindAction(m_Cfg.LeftClickAction));
-            m_MiddleClickAction = InputActionReference.Create(m_InputActionAsset.FindAction(m_Cfg.MiddleClickAction));
-            m_RightClickAction = InputActionReference.Create(m_InputActionAsset.FindAction(m_Cfg.RightClickAction));
-            m_ScrollWheelAction = InputActionReference.Create(m_InputActionAsset.FindAction(m_Cfg.ScrollWheelAction));
-
-            if (m_PointAction.action != null)
+            if (m_InputActionAsset != null)
```

Comment:
I believe this just addresses the null checking. There is also a need to monitor any changes to project wide actions in run-time and reconfigure based on that or the concept needs modified to be more strict to not allow changes during run-time but then its an editor only feature and would basically imply that a setter on InputSystem.actions shouldn't exist.